### PR TITLE
Correct chat id for Telegram action

### PR DESCRIPTION
**Pull request by mwl**
With `chat.id` it's also possible to add the bot to a group

### CODE REVIEW

**Comment by Confectrician** [comment]
Hello and thanks for your contribution! 👋 
Many parts of the documentation are [imported from other openHAB repositories](https://github.com/openhab/openhab-docs#automatically-generated-parts), e.g., the Binding- or add-on specific repository. This way we ensure that the articles are always in sync with the actual source code. This method has proven to be quite helpful to keep documentation up-to-date.
Long story short, you'll need to propose your change in the correct repository: 
https://github.com/openhab/openhab1-addons/blob/master/bundles/action/org.openhab.action.telegram/README.md

Thanks again for your willingness to contribute! 🙇
Please close this PR after you've created the new one.

**Comment by mwl** [comment]
Sorry, I didn't notice that. I've updated the README you referred me to.
